force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +45,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        if (age>=18){
```

Comment:
Be careful of the formatting of your brackets. The two most accepted styles for apex are https://en.wikipedia.org/wiki/Indentation_style#Allman_style and https://en.wikipedia.org/wiki/Indentation_style#K&R The second is more popular in Java, and thus the typical style for Apex